spec/javascripts/support/console.js

```diff
@@ -44,15 +44,17 @@ export function chaiConsoleSpy(chai, utils) {
  * `chaiConsoleSpy` Chai plugin.
  */
 export function useConsoleLogSpy() {
+  let originalConsoleError;
   beforeEach(() => {
     console.unverifiedCalls = [];
-    sinon.stub(console, 'error').callsFake((message, ...args) => {
+    originalConsoleError = console.error;
+    console.error = sinon.stub().callsFake((message, ...args) => {
       console.unverifiedCalls = console.unverifiedCalls.concat(format(message, ...args));
     });
   });
 
   afterEach(() => {
-    console.error.restore();
+    console.error = originalConsoleError;
```

Comment:
I didn't mean to include the changes here, but they are necessary for an upgrade to [`@testing-libary/react-hooks`](https://www.npmjs.com/package/@testing-library/react-hooks), which I had temporarily updated in a misguided effort to resolve a failing spec. I guess I'll just leave it, since we'll need it for that upgrade anyways.